com.djoffe.gnukem.json

```
@@ -0,0 +1,62 @@
+{
+  "app-id": "com.djoffe.gnukem",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "davegnukem",
+  "rename-desktop-file": "davegnukem.desktop",
+  "rename-icon-file": "davegnukem",
```

```suggestion
  "rename-icon": "davegnukem",
```

I said the key didn't exist.